include/kickcat/Frame.h

```diff
@@ -49,21 +49,25 @@ namespace kickcat
 
         /// \return true if datagram can be extracted, false otherwise
         bool isDatagramAvailable() const { return is_datagram_available_; }
+        void setIsDatagramAvailable(bool value) { is_datagram_available_ = value; }
```

Comment:
You may force this accessor to always set the flag to true since there is no other needs.